examples/jsm/loaders/LDrawLoader.js

```diff
@@ -2080,8 +2102,24 @@ class LDrawLoader extends Loader {
 				material = loader.getMaterial( colorCode );
 				if ( material === null ) {
 
-					// otherwise throw an error if this is final opportunity to set the material
-					throw new Error( `LDrawLoader: Material properties for code ${ colorCode } not available.` );
+					// otherwise throw a warning if this is final opportunity to set the material
+					console.warn( `LDrawLoader: Material properties for code ${ colorCode } not available.` );
+
+					// And return the 'missing color' material
+					material = loader.missingColorMaterial;
+					material.name = "Missing material for color code '" + colorCode + "'";
+					material.userData.code = colorCode;
```

Comment:
So we return the same material but change its name and userData in every invocation?... If so, I think it should be cloned.

Reply:
You're right. @gkjohnson What do you think is better, to clone or to remove the name modification?

Reply:
I've commited the change to generic missing material without clone(). Let me know if you agree.

Reply:
👍

Reply:
I think this is good! The warning should be enough to let users know which information is missing.